Makefile

```diff
@@ -176,6 +176,24 @@ graplctl: ## Build graplctl and install it to the project root
 build-ux: ## Build website assets
 	cd src/js/engagement_view && yarn install && yarn build
 
+# Create the dist directory if necessary; don't need to document this
+# formally for `make help`, though.
+dist:
+	mkdir dist
```

Comment:
I think you need the "oh it's okay if dist already exists" option `--parents` (poorly named) on this

mkdir __pycache__
mkdir: cannot create directory ‘__pycache__’: File exists
echo $?
1

Reply:
Make takes care of this. `dist` is not a phony target, so Make treats this as a recipe for how to "build" `dist`.